src/components/scenes/SwapSettingsScene.js

```diff
@@ -181,7 +181,7 @@ export class SwapSettings extends Component<Props, State> {
 
 const getStyles = cacheStyles(theme => ({
   instructionArea: {
-    backgroundColor: theme.settingsHeaderRowBackground,
+    backgroundColor: theme.listHeaderBackground,
```

Comment:
should use settingsRowSubHeader = palette.transparent. Ask designer what design should be.